Source/WebCore/platform/graphics/haiku/ImageBufferHaiku.cpp

```diff
@@ -32,8 +32,8 @@
 #include "MIMETypeRegistry.h"
 #include "NotImplemented.h"
 #include "StillImageHaiku.h"
-#include "JavaScriptCore/GenericTypedArrayViewInlines.h"
-#include "JavaScriptCore/JSGenericTypedArrayView.h"
+#include "JavaScriptCore/JSCInlines.h"
```

Comment:
Have you checked that the changes in WebCore do not build the WebKitLegacy build?

Reply:
Changing webcore doesnt build webkitlegacy? i thought all the linked files would be rebuilt again not sure let me build again and let you know 😄

Reply:
you can build DumpRenderTree or HaikuLauncher (ninja HaikuLauncher) to test building just the webkitlegacy parts. Since WebKit2 build is broken the build may stop before or after building webkit...

Reply:
This change is absolutely safe Pulkomandy 😄